src/main/resources/templates/bo/form-request.html

```
@@ -25,12 +25,12 @@ <h2 class="text-lg font-bold text-slate-700">폼 변환 요청</h2>
         <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 mb-4">
             <div class="lg:col-span-2">
                 <label class="block text-xs font-bold text-slate-500 mb-1">폼 링크 <span class="text-red-500">*</span></label>
-                <input type="url" id="newFormLink" placeholder="https://docs.google.com/forms/d/e/..."
+                <input type="url" id="newFormLink" placeholder="https://docs.google.com/forms/d/.../edit" required
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)